pkg/install/install.go

```diff
@@ -1,940 +0,0 @@
-			l.Error("Could not connect to Kubernetes. " +
-				"Make sure Kubernetes is running and is accessible from this computer/server.")
-		}
-		return nil, err
-	}
-	cli.kubeClient = k
-	cli.versionService = versionservice.New(c.VersionMetadataURL)
-	return cli, nil
-}
-
-// Run runs the operators installation process.
-func (o *Install) Run(ctx context.Context) error { //nolint:funlen
-	// TODO: we shall probably split this into "install" and "add namespaces"
-	// Otherwise the logic is hard to maintain - we need to make sure not to,
-	// for example, install a different version of operators per namespace, if
-	// we are always installing the "latest" version.
-	if err := o.populateConfig(); err != nil {
-		return err
-	}
-
-	var err error
-	installSteps := []common.Step{}
-
-	meta, err := o.versionService.GetEverestMetadata(ctx)
-	if err != nil {
-		return errors.Join(err, errors.New("could not fetch version metadata"))
-	}
-	latest, latestMeta, err := o.latestVersion(meta)
-	if err != nil {
-		return err
-	}
-
-	if err = o.checkRequirements(latestMeta); err != nil {
-		return err
-	}
-
-	recVer, err := version.RecommendedVersions(latestMeta)
-	if err != nil {
-		return err
-	}
-	if recVer.EverestOperator == nil {
-		// If there's no recommended version of the operator, install the same version as Everest.
-		recVer.EverestOperator = latest
-	}
-
-	o.l.Debugf("Everest latest version available: %s", latest)
-	o.l.Debugf("Everest version information %#v", latestMeta)
-
-	if o.config.SkipEnvDetection {
-		o.l.Debug("Skipping Kubernetes environment detection")
-	} else {
-		if err := o.detectEnvironment(ctx); err != nil {
-			return fmt.Errorf("could not detect Kubernetes environment: %w", err)
-		}
-	}
-
-	if o.config.SkipOLM {
-		o.l.Debug("Skipping OLM installation")
-	} else {
-		installSteps = append(installSteps, o.provisionOLM()...)
-	}
-
-	installSteps = append(installSteps, o.installCatalog(latest)...)
-	installSteps = append(installSteps, o.provisionMonitoringStack()...)
-	installSteps = append(installSteps, o.provisionEverestComponents(latest, recVer)...)
-
-	var out io.Writer = os.Stdout
-	if !o.config.Pretty {
-		out = io.Discard
-	}
-	fmt.Fprintln(out, output.Info("Installing Everest version %s", latest))
-	if err := common.RunStepsWithSpinner(ctx, installSteps, out); err != nil {
-		return err
-	}
-	fmt.Fprint(os.Stdout, "\n", output.Rocket(postInstallMessage))
-
-	isAdminSecure, err := o.kubeClient.Accounts().IsSecure(ctx, common.EverestAdminUser)
-	if err != nil {
-		return errors.Join(err, errors.New("could not check if the admin password is secure"))
-	}
-	if !isAdminSecure {
-		fmt.Fprint(os.Stdout, "\n", common.InitialPasswordWarningMessage)
-	}
-	return nil
-}
-
-func (o *Install) checkRequirements(meta *versionpb.MetadataVersion) error {
-	supVer, err := common.NewSupportedVersion(meta)
-	if err != nil {
-		return err
-	}
-
-	if err := common.CheckK8sRequirements(supVer, o.l, o.kubeClient); err != nil {
-		return err
-	}
```

Comment:
Just to confirm, now the k8s version requirement is baked into the chart itself so we no longer need to specify it in the version service, hence the removal of this check, correct?

Reply:
Yes, correct. Helm checks it for us

This is where it is specified - https://github.com/percona/percona-helm-charts/blob/main/charts/everest/Chart.yaml#L7